pandas/core/generic.py

```diff
@@ -666,6 +666,7 @@ def size(self) -> int:
         See Also
         --------
         ndarray.size : Number of elements in the array.
+            (`ndarray.size <https://numpy.org/doc/stable/reference/generated/numpy.ndarray.size.html>`_)
```

Comment:
Instead if you add a `numpy.` in front of `ndarray.size` it should hyperlink

Reply:
Done! I didn't know it was that simple, but once I looked into the Sphinx documentation, it makes sense.